components/Video.ts

```diff
@@ -0,0 +1,30 @@
+import { jwt } from 'twilio';
+import { connect } from 'twilio-video';
+
+const getRandomIntInclusive = (min, max) => {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+};
+
+const generateToken = () => {
+  const twilioAccountSid = process.env.TWILIO_SID;
+  const twilioApiKey = process.env.TWILIO_API_KEY;
+  const twilioApiSecret = process.env.TWILIO_API_SECRET;
+  const token = new jwt.AccessToken(
+    twilioAccountSid,
+    twilioApiKey,
+    twilioApiSecret
+  );
```

Comment:
```suggestion
  const { TWILIO_SID, TWILIO_API_KEY, TWILIO_API_SECRET } = process.env;
  const token = new jwt.AccessToken(
    TWILIO_SID,
    TWILIO_API_KEY,
    TWILIO_API_SECRET,
  );
```

Also added a trailing comma in the args, that way if an additional one needs to be added for any reason, the diff won't have to touch the previous line!

Reply:
Usually I love the trailing comma, but Prettier removed the trailing comma when I added it 😬

Reply:
Great call on the destructuring of `process.env`.